utils/maintenance/GitHub.php

```diff
@@ -281,7 +289,7 @@ public static function request(
 
 		$response = Utils\http_request( $verb, $url, $args, $headers );
 
-		if ( 20 != substr( $response->status_code, 0, 2 ) ) {
+		if ( 20 !== absint( substr( $response->status_code, 0, 2 ) ) ) {
```

Comment:
Just wondering: Why did you choose to use `absint()` here rather than a simple type cast ?